front-end/src/components/main-page/MainPage.jsx

```diff
@@ -6,6 +6,7 @@ import BaseButton from "../general/BaseButton.jsx";
 import TranscriptUploadModal from "./TranscriptUploadModal.jsx";
 import Title from "../icons/title.jsx";
 import {exampleTranscript} from "../helpers/ExampleTranscript.js";
+import TranscriptDescription from "./TranscriptDescription"
```

Comment:
Can we have the file extension here for consistency? I know it's not necessary but we have it for import statements everywhere else.
```suggestion
import TranscriptDescription from "./TranscriptDescription.jsx"
```